include/fty-srr-rest.h

```diff
@@ -31,6 +31,6 @@
 #define AGENT_NAME                      "fty-srr-rest"
 #define AGENT_NAME_REQUEST_DESTINATION  "fty-srr"
 #define MSG_QUEUE_NAME                  "ETN.Q.IPMCORE.SRR"
-#define DEFAULT_TIME_OUT                70
```

Comment:
600 would be better

Reply:
I hesitated with 10mn, considering that it's the UI timeout. But seemed to me a bit high, and I'm not enough versed in SRR (1rst contact today) to be sure... I'll amend

Reply:
It's more than a UI timeout, tntnet will hard-reboot (and kill all sessions) if it detects a request is still ongoing after 10 minutes last time I've checked. This timeout must be less than tntnet's own request watchdog timeout with a suitable margin.